networking/networking.go

```diff
@@ -116,6 +125,54 @@ func Setup(podRoot string, podID types.UUID, fps []ForwardedPort, netList common
 	return &n, nil
 }
 
+// enableDefaultLocalnetRouting enables the route_localnet attribute on the supposedly default network interface.
+// This allows to setup a loopback NAT to access port forwardings to Pods on the host with the localhost address.
+func (n *Networking) enableDefaultLocalnetRouting() error {
+	routeLocalnetFormat := ""
+
+	defaultHostIP, err := n.GetDefaultHostIP()
+	if err != nil {
+		return err
+	}
+
+	defaultHostIPstring := defaultHostIP.String()
+	switch {
+	case strings.Contains(defaultHostIPstring, "."):
+		routeLocalnetFormat = "/proc/sys/net/ipv4/conf/%s/route_localnet"
+	case strings.Contains(defaultHostIPstring, ":"):
+		return fmt.Errorf("unexpected IPv6 Address returned for default host interface: %q", defaultHostIPstring)
+	default:
+		return fmt.Errorf("unknown type for default Host IP: %q", defaultHostIPstring)
+	}
+
+	hostIfaces, err := n.GetIfacesByIP(defaultHostIP)
+	if err != nil {
+		return err
+	}
+
+	for _, hostIface := range hostIfaces {
+
+		routeLocalnetPath := fmt.Sprintf(routeLocalnetFormat, hostIface.Name)
+		routeLocalnetValue, err := ioutil.ReadFile(routeLocalnetPath)
+		if err != nil {
+			return err
+		}
+		if string(routeLocalnetValue) != "1" {
```

Comment:
I guess it is ok to write "1" if it is already set to "1"? If so, no need to read it before and just write it. That would be less code.